pkg/controlplane/instance.go

```diff
@@ -510,6 +512,23 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 			go controller.Run(hookContext.StopCh)
 			return nil
 		})
+		// Labels for apiserver idenitiy leases switched from k8s.io/component=kube-apiserver to apiserver.kubernetes.io/identity=kube-apiserver.
+		// For compatibility, garbage collect leases with both labels for at least 1 release
+		// TODO: remove in Kubernetes 1.28
```

Comment:
```suggestion
		// Labels for apiserver idenitiy leases switched from k8s.io/component=kube-apiserver (v1.26) to apiserver.kubernetes.io/identity=kube-apiserver (v1.27).
		// For compatibility, garbage collect leases with both labels for at least two releases (26 + 2 = 28)
		// TODO: remove in Kubernetes 1.28
```

Remind me: are we backporting this change?

Reply:
Will defer to SIG leads on whether the backport is worthwhile.